### PR TITLE
Bugfix/NextJS support

### PR DESCRIPTION
**Pull request by cesarenaldi**
There was a misconfiguration with the `@lens-protocol/react` `exports` which was causing some bundler to not find the `@lens-protocol/react/web` entry point. NextJS SWC configuration was one of these.

This PR fixes the issue and adds a NextJS example app which will be included into the CI build step preventing similar issues to happen in the future.

### CODE REVIEW

File: packages/react/package.json [hunk 1]
```diff
@@ -21,12 +21,12 @@
...
-        "types": "./dist/esm/web.d.ts",
-        "default": "./dist/esm/web.js"
+        "types": "./dist/esm/web/index.d.ts",
+        "default": "./dist/esm/web/index.js"
       },
       "require": {
-        "types": "./dist/cjs/web.d.ts",
-        "default": "./dist/cjs/web.js"
+        "types": "./dist/cjs/web/index.d.ts",
+        "default": "./dist/cjs/web/index.js"
```
Comment (krzysu) [inline packages/react/package.json:29]: so this is the actual fix only?

**Comment by krzysu** [comment]
we don't need files from `examples/nextjs/public` folder, the rest LGTM!

**Review by krzysu — APPROVED** [review]

File: packages/react/package.json (hunk 1 above)
Comment (cesarenaldi) [reply packages/react/package.json:29]: Fixed here: https://github.com/lens-protocol/lens-sdk/pull/51/commits/42307333ca23343f5dd642871a1c12c5f3cc1d8f

**Comment by cesarenaldi** [comment]
> we don't need files from `examples/nextjs/public` folder, the rest LGTM!

Fixed here: https://github.com/lens-protocol/lens-sdk/pull/51/commits/42307333ca23343f5dd642871a1c12c5f3cc1d8f

**Pull request merged**